### DISCUSSION TITLE
Navigation Instant Not Working on Prod (Build)

### DISCUSSION THREAD

**Discussion by Akkadius** [squidfunk/mkdocs-material#3238] [Q&A] [answered]
Gitbook recently made a terrible update so I'm moving my communities docs over to mkdocs again. I've used mkdocs in the past and actually contributed a few minor changes in the past, thank you for your work.

Repo and instant setting https://github.com/EQEmu/eqemu-docs-v2/blob/main/mkdocs.yml#L17

Instant works when running `mkdocs serve` but running it through a web server with a configuration made to run SPA configurations does full page loads every time

http://docsv2.eqemu.io/

Any ideas?

Thanks!

**Comment by Akkadius** [comment, marked as answer]
@squidfunk I'm guessing this might be something you'd know more immediately

I did Google quite a bit before dropping this here. Thanks!

**Reply by squidfunk** [reply]
Try adding [`site_url`](https://github.com/squidfunk/mkdocs-material/blob/77171e02a4aa4b346cd95f796f5721369d02ab8a/mkdocs.yml#L23), might be related.

**Reply by Akkadius** [reply]
That was it!! Thank you so much!
